fxgl-samples/src/main/java/sandbox/tiled/TiledBenchmarkSample.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * FXGL - JavaFX Game Library. The MIT License (MIT).
+ * Copyright (c) AlmasB (almaslvl@gmail.com).
+ * See LICENSE for details.
+ */
+
+package sandbox.tiled;
+
+import com.almasb.fxgl.app.GameApplication;
+import com.almasb.fxgl.app.GameSettings;
+import com.almasb.fxgl.entity.Entity;
+import com.almasb.fxgl.entity.EntityFactory;
+import com.almasb.fxgl.entity.SpawnData;
+import com.almasb.fxgl.entity.Spawns;
+
+import static com.almasb.fxgl.dsl.FXGL.*;
+
+/**
+ * This is a sample for testing the performance of tiled map loading
+ *
+ * @author Adam Bocco (adam.bocco) (adam.bocco@gmail.com)
+ */
+public class TiledBenchmarkSample extends GameApplication {
+
+    @Override
+    protected void initSettings(GameSettings settings) {
+
+        settings.setWidth(500);
+        settings.setHeight(500);
+        settings.setTitle("TiledBenchmarkSample");
+        settings.setVersion("0.1");
+    }
+
+    @Override
+    protected void initGame() {
+        getGameWorld().addEntityFactory(new TiledObjectFactory());
+
+        var t1engine = getEngineTimer().getNow();
+        var t1sys = System.nanoTime();
+
+        setLevelFromMap("tmx/benchmarking/csv_1layer_noflip_500x500.tmx");

```

Comment:
Set level from map can give misleading times as it also includes the call to `gameworld.setLevel`, which in turn forces a scene graph update. I also made this mistake too at first :)

Can you replace `setLevelFromMap` to just load the level. You can check the Tiled sample from sandbox/benchmark.

Incidentally, since all benchmark code is in there, perhaps we can move this file there too. I'm thinking then `sandbox/tiled` is for developing Tiled related features. See what you think

Reply:
I made those fixes and created a simple UI to run the tests although I'm not sure if that was wise. Would that interfere much with the times for the tests themselves even if on separate threads? 

 I didn't notice the place for benchmarking samples earlier. I left the empty tiled package in sandbox as well, I can see myself adding something to there in the future.

Reply:
Haven't checked yet, but if you start the benchmark tests using `getExecutor().startAsync()`, it should be fine. Essentially, any thread, apart from UI thread, is fine.